Merkll/Quick-Credit
# #165840678 Convert commonjs require to es6 import

#### What does this PR do?
convert non es6 require and export to es6 import and export and also refactor the codebase
#### Description of Task to be completed?
es6 import and export replace commonjs
#### How should this be manually tested?
```Clone this repo```
Run ```npm install && npm test```
All test cases should pass
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
 #165840678
